### DISCUSSION TITLE
Any early signal on what it'd take to support 7.0?

### DISCUSSION THREAD

**Discussion by walkerdb** [storybookjs/react-native#436] [General]
Super excited for the soon-to-come official 6.0 support for react-native, really appreciate all the work that's gone into it!

Given core storybook has been ramping up the hype recently for their upcoming 7.0 release is there any early indication on the amount of additional work it would take to support 7.0 here as well?

**Comment by dannyhw** [comment]
@walkerdb hey, glad your excited (me too 😀). 

Yeah the main work is getting the on demand story store working, currently we still use a v6 style story store. However with the work to support 6.4+ (where the v7 store got introduced) much of the ground work is done. Not to say it will be easy, but it shouldn't be as big of a task as v6 was.
